src/main/java/frc/robot/config/Config.java

```diff
@@ -215,8 +215,14 @@ public static class CANID {
             new Translation2d(-kWheelBase / 2, kTrackWidth / 2),
             new Translation2d(-kWheelBase / 2, -kTrackWidth / 2));
 
-        public static final double teleopSpeed = 3.0;
-        public static final double kMaxTeleopAngularSpeed = Math.PI*2.5;
+        public static final double teleopFastSpeed = 3.0;
+        public static final double teleopFastAngularSpeed = Math.PI*3.0;
+
+        public static final double teleopSlowSpeed = 0.3;
+        public static final double teleopSlowAngularSpeed = 0.3;
```

Comment:
The slow speed will likely need to be tuned once we have the comp robot and are trying to align to nodes.